### PR TITLE
Bloop shoot

### PR DESCRIPTION
**Pull request by bengrif**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/manipulator/ShooterSubsystem.java [hunk 1]
```diff
@@ -15,7 +15,7 @@
 import java.util.function.DoubleSupplier;
 
 public class ShooterSubsystem extends SubsystemBase {
-  private final boolean SHUFFLEBOARD_ENABLED = false;
+  private final boolean SHUFFLEBOARD_ENABLED = true;
```
Comment (Kellenn) [inline src/main/java/frc/robot/subsystems/manipulator/ShooterSubsystem.java:18]: This can be false.

**Review by Kellenn — COMMENTED** [review]
What is a bloop shoot?

File: src/main/java/frc/robot/subsystems/manipulator/ShooterSubsystem.java (hunk 1 above)
Comment (bengrif) [reply src/main/java/frc/robot/subsystems/manipulator/ShooterSubsystem.java:18]: OK, I was just trying to get the telemetry to monitor it.

**Comment by bengrif** [comment]
A Bloop Shot is a shot not strong enough to score, but strong enough to eject the note safely.   No sounds are generated, sorry.

**Pull request merged**